smash/core/simulation/estimate/estimate.py

```diff
@@ -125,13 +243,24 @@ def _multiset_estimate(
     else:  # Should be unreachable
         pass
 
-    # TODO: add return options here
-    estimator(
+    ret_forward_run, lcurve_multiset = estimator(
         alpha,
         model,
-        multiset._cost_options,
         prior_data,
         density,
         multiset.cost,
-        common_options["ncpu"],
+        multiset._cost_options,
+        common_options,
+        return_options,
     )
+
+    fret = {} if ret_forward_run is None else ret_forward_run.__dict__
+    pyret = {}
+
+    if "lcurve_multiset" in return_options["keys"]:
+        pyret["lcurve_multiset"] = lcurve_multiset
```

Comment:
Juste pour le style ;) (j'ai fait black pour ça)
```suggestion
    pyret = (
        {"lcurve_multiset": lcurve_multiset}
        if "lcurve_multiset" in return_options["keys"]
        else {}
    )
```